.env.template

```diff
@@ -2,4 +2,7 @@ INFURA_PROJECT_ID=<infura_project_id>
 INFURA_PROJECT_SECRET=<infura_secret_id>
 WALLET_PUBLIC_ADDRESS=<wallet_public_address>
 WALLET_PRIVATE_KEY=<wallet_private_key>
-EVM_RPC_URL=<goerli_rpc_url>
\ No newline at end of file
+EVM_RPC_URL=<goerli_rpc_url>
+INFURA_IPFS_URL=<infura_ipfs_url>
+INFURA_IPFS_PROJECT_ID=<infura_ipfs_project_id>
+INFURA_IPFS_PROJECT_SECRET=<infura_ipfs_project_secret>
```

Comment:
Should we make this service agnostic? e.g., being able to use something else than infura IPFS service (like pinata or others)?